src/intl/en/glossary.json

```diff
@@ -0,0 +1,322 @@
+{
+  "51%-attack-term": "51% attack",
+  "51%-attack-definition": "A type of attack on a decentralized <a href=\"#network\">network</a> where a group gains control of the majority of <a href=\"#node\">nodes</a>. This would allow them to defraud the blockchain by reversing <a href=\"#transactions\">transactions</a> and double spending <a href=\"ether\">ether</a> and other tokens.",
```

Comment:
Noting that a lot of definitions have links inside. Most of those links are pointing to other definitions like `/#transactions` but those only exist on the glossary page, so they will not work when used inside the tooltips.

Maybe we could modify those links to always point to the glossary page.
For example, `#transactions` => should be transformed to `/glossary/#transactions`

Reply:
Good catch, yeah will update this